packages/core/components/GlobalStyle/GlobalStyle.tsx

```diff
@@ -66,7 +29,7 @@ const GlobalStyle = createGlobalStyle`
     src: url('${videowoff2}') format('woff2'),
          url('${videowoff}') format('woff'),
          url('${videottf}') format('truetype'),
-         url('${videoeot}?#iefix') format('embedded-opentype');
+         url('${videoeot}.eot?#iefix') format('embedded-opentype');
```

Comment:
what do you think to remove this change?

Reply:
I couldn't agree more!